autoscale_cloudcafe/autoscale/client.py

```diff
@@ -940,12 +941,17 @@ def get_node_info_detail(self, pool_id, node_id):
         return self.request('GET', url,
                             response_entity_type=RackConnectLBNodeDetail)
 
-    def remove_node_from_pool(self, pool_id, node_id, requestslib_kwargs=None):
+    def remove_server_from_pool(self, pool_id, server_id,
+                                requestslib_kwargs=None):
         """
-        :summary: Remove a node from a lb pool
+        :summary: Remove a server from a lb pool using the bulk delete API
         """
-        url = self.url + '/load_balancer_pools/{0}/nodes/{1}'.format(pool_id,
```

Comment:
Any specific reason for changing the implementation? I don't know if having bulk delete makes a difference w.r.t otter.